logging/logging.go

```diff
@@ -191,6 +195,13 @@ func NewClient(ctx context.Context, parent string, opts ...option.ClientOption)
 
 func makeParent(parent string) (string, error) {
 	if !strings.ContainsRune(parent, '/') {
+		if parent == "" {
```

Comment:
Instead of using empty string, you can consider using a sentinel similar to [Datastore](https://github.com/googleapis/google-cloud-go/blob/afe7c98036c198995075530d4228f1f4ae3f1222/datastore/datastore.go#L109-L145) and [Firestore](https://github.com/googleapis/google-cloud-go/blob/afe7c98036c198995075530d4228f1f4ae3f1222/firestore/client.go#L92-L107). BigQuery and PubSub also use sentinels.

Reply:
Done.